Assets/RGLUnityPlugin/Scripts/SceneManager.cs

```diff
@@ -129,6 +136,12 @@ public void DoUpdate()
             toAddGOs.ExceptWith(lastFrameGameObjects);
             RGLObject[] toAdd = IntoRGLObjects(toAddGOs).ToArray();
 
+            if(textureReading)
+            {
+                // Add textures
+                AddTextures(toAdd);
```

Comment:
1. ![image](https://github.com/tier4/AWSIM/assets/112629916/4a4d4200-8dcd-4c36-ba9a-1820aa880725)
2. Add a profiler to this method.